CHANGELOG.md

```diff
@@ -1,10 +1,26 @@
-# Unreleased
+# Changelog
+All notable changes to this project will be documented in this file.
 
-# v0.2.0
-* Provider is now one file, for easier distribution
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-# v0.1.1
-* Removed a debug statement
+## Unreleased
 
-# v0.1.0
-* initial release
\ No newline at end of file
+## [0.2.1] -  2019-06-29
+
+## [0.2.0] - 2015-06-29
+### Changed
+- Provider is now one file, for easier distribution
+
+## [0.1.1] - 2015-06-29
+### Removed
+- Removed a debug statement
+
+## [0.1.0] - 2015-06-29
+### Added
+- initial release
+
+[0.2.1]: https://github.com/cyberark/summon-keyring/releases/tag/v0.2.1
```

Comment:
When I run the parse-changelog.sh script locally, it complains about these reference lines. I had to use these lines:
```
[Unreleased]: https://github.com/cyberark/summon-keyring/compare/v0.2.1...HEAD
[0.2.1]: https://github.com/cyberark/summon-keyring/compare/v0.2.0...v0.2.1
[0.2.0]: https://github.com/cyberark/summon-keyring/compare/v0.1.1...v0.2.0
[0.1.1]: https://github.com/cyberark/summon-keyring/compare/v0.1.0...v0.1.1
[0.1.0]: https://github.com/cyberark/summon-keyring/compare/v0.1.0...v0.1.0
```